core/docker/README.md

```diff
@@ -76,6 +76,25 @@ The Docker build process will print the ID of the image, which will also
 be tagged with `trino:xxx-SNAPSHOT`, where `xxx-SNAPSHOT` is the version
 number of the Trino Maven build.
 
+To build an image for a specific released version of Trino,
```

Comment:
It feels like it should be easier for end-users to discover this section. @mosabua WDYT about adding a docs section about the Docker image? It's a pretty popular deployment method and currently, there's no mention of it in the docs.

Reply:
I very much would like to have such a section .. same for the RPM and the Helm chart .. I am hoping that @nineinchnick @colebow and @bitsondatadev and take that on as a bit of a project soon. We do have a bunch of docs  that can be lifted and shifted for this..